Editor/SettingsAnalysis/SrpAssetSettingsAnalyzer.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections.Generic;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Diagnostic;
+using Unity.ProjectAuditor.Editor.Modules;
+using UnityEngine;
+using UnityEngine.Rendering;
+#if PACKAGE_URP
+using UnityEngine.Rendering.Universal;
+#elif PACKAGE_HDRP
+using System.Reflection;
+using UnityEngine.Rendering.HighDefinition;
+#endif
+
+namespace Unity.ProjectAuditor.Editor.SettingsAnalysis
+{
+    class SrpAssetSettingsAnalyzer : ISettingsModuleAnalyzer
+    {
+        internal const string PAS1008 = nameof(PAS1008);
+
+        static readonly Descriptor k_SRPBatcherSettingDescriptor = new Descriptor(
+            PAS1008,
+            "SRP Asset: SRP Batcher",
+            Area.CPU,
+            "SRP batcher is disabled in Render Pipeline Asset.",
+            "Enable SRP batcher in Render Pipeline Asset. This will reduce the CPU time Unity requires to prepare and dispatch draw calls for materials that use the same shader variant.")
+        {
+            messageFormat = "SRP batcher is disabled in '{0}' in '{1}'",
```

Comment:
I just wondered what we'd see here.

It is hard to guess if the message that comes out of it reads correctly. ;)

Reply:
It will be:
1. For default render pipeline: SRP batcher is disabled in FileName in Default Rendering Pipeline Asset
2. For quality render pipeline: SRP batcher is disabled in FileName in Rendering Pipeline Asset in Quality Level: High

Reply:
Check screenshot in the description